hashicorp/dev-portal
# fix: add Consul ConfigEntryReference component

## "Ready for Review" checklist




---

## Relevant links



- [Preview link](https://dev-portal-git-dsconsul-configentryreference-hashicorp.vercel.app/docs/connect/config-entries/ingress-gateway#available-fields) 🔎
- [Asana task](url) 🎟️

## What

Adds Consul's custom `ConfigEntryReference` component

## Why

The `ConfigEntryReference` component used [here](https://consul-pbqzi3dcm-hashicorp.vercel.app/docs/connect/config-entries/ingress-gateway#available-fields) was missed during the migration.

## How

## Testing

- Visit [this URL](https://dev-portal-git-dsconsul-configentryreference-hashicorp.vercel.app/)
- Choose Consul in the product switcher in the bottom left
- Go to [this URL](https://dev-portal-git-dsconsul-configentryreference-hashicorp.vercel.app/docs/connect/config-entries/ingress-gateway#available-fields)
- Scroll down and confirm that "Available Fields" section matches [this URL](https://consul-pbqzi3dcm-hashicorp.vercel.app/docs/connect/config-entries/ingress-gateway#available-fields).

## Anything else?
